libs/cardano-data/test/Test/Data/UMap.hs

```diff
@@ -0,0 +1,474 @@
+      testPropertyN 1000 "bisimulation Ptr map" bisimulatePtr,
+      testPropertyN 1000 "next iterates over Rewards" iterRew,
+      testPropertyN 1000 "next iterates over Delegtions" iterDel,
+      testPropertyN 1000 "next iterates over Ptrs" iterPtr,
+      testPropertyN 1000 "leastUpperBound Rewards" lubRew,
+      testPropertyN 1000 "leastUpperBound Delegations" lubDel,
+      testPropertyN 1000 "leastUpperBound Ptrs" lubPtr,
+      testPropertyN 1000 "delete Rewards" delRew,
+      testPropertyN 1000 "delete Delegations" delDel,
+      testPropertyN 1000 "delete Ptrs" delPtr,
+      testPropertyN 1000 "insertWith (<>) Rewards" insertRew,
+      testPropertyN 1000 "insertWith (\\ old new -> new) Delegations" insertDel,
+      testPropertyN 1000 "insertWith (\\ old new -> old) Ptrs" insertPtr,
+      testPropertyN 1000 "lookup Rewards" lookRew,
+      testPropertyN 1000 "lookup Delegations" lookDel,
+      testPropertyN 1000 "lookup Ptrs" lookPtr,
+      testPropertyN 1000 "null Rewards" nullRew,
+      testPropertyN 1000 "null Delegations" nullDel,
+      testPropertyN 1000 "null Ptrs" nullPtr,
+      testPropertyN 1000 "domain  Rewards" domRew,
+      testPropertyN 1000 "domain Delegations" domDel,
+      testPropertyN 1000 "domain Ptrs" domPtr,
+      testPropertyN 1000 "range  Rewards" rngRew,
+      testPropertyN 1000 "range Delegations" rngDel,
+      testPropertyN 1000 "range Ptrs" rngPtr,
+      testPropertyN 1000 "left biased (x  ∪ (k,v)) Rewards" uLeftRew,
+      testPropertyN 1000 "left biased (x  ∪ (k,v)) Delegations" uLeftDel,
+      testPropertyN 1000 "left biased (x  ∪ (k,v)) Ptrs" uLeftPtr,
+      testPropertyN 1000 "right biased (x ⨃ y) Rewards" uRightRew,
+      testPropertyN 1000 "right biased (x ⨃ y) Delegations" uRightDel,
+      testPropertyN 1000 "right biased (x ⨃ y) Ptrs" uRightPtr,
+      testPropertyN 1000 "monoidal sum (x  ∪+ y) Ptrs" uSumRew,
+      testPropertyN 1000 "domain exclude (x ⋪ y) Rewards" domExRew,
+      testPropertyN 1000 "domain exclude (x ⋪ y) Delegations" domExDel,
+      testPropertyN 1000 "domain exclude (x ⋪ y) Ptrs" domExPtr,
+      testPropertyN 1000 "range exclude (x  ⋫ y) Rewards" rngExRew,
+      testPropertyN 1000 "range exclude (x  ⋫ y) Delegations" rngExDel,
+      testPropertyN 1000 "range exclude (x  ⋫ y) Ptrs" rngExPtr,
+      testPropertyN 1000 "member Rewards" memberRew,
+      testPropertyN 1000 "member Delegations" memberDel,
+      testPropertyN 1000 "member Ptrs" memberPtr,
+      testPropertyN 1000 "domain restrict Rewards" domRestrictRew,
+      testPropertyN 1000 "domain restrict Delegations" domRestrictDel,
+      testPropertyN 1000 "domain restrict Ptrs" domRestrictPtr,
+      testPropertyN 1000 "unify unUnify roundtrip" unifyRoundTrip1,
+      testPropertyN 1000 "unUnify unify roundtrip" unifyRoundTrip2,
+      testPropertyN 1000 "size is accurate" sizeUMap
+    ]
+
+-- =======================================================
+
+test :: IO ()
+test = defaultMain alltests
```

Comment:
Its is only dead code, if your not trying to run the tests from inside the repl

Reply:
I realize that, but if you are the only one running it from repl it's not that hard for you to run `> defaultMain alltests` instead of `> test`, right?

Reply:
I do not agree with this, but I have removed it in order to get things moving. I think that every test file should have 
'test = defaultMain alltests'
This is extremely useful in a debugging workflow, and I don't think it is dead code at all, It is all a matter of perspective.